chainer_compiler/chainer_compiler.py

```diff
@@ -262,6 +260,8 @@ def compile(self, onnx_file):
                              bwd_graph.dump() +
                              '\n=== ^^^ backward ^^^ ===\n')
 
+        _chainer_compiler_core.configure(skip_inference=True)
```

Comment:
I think the original logic,
```
        compiler_kwargs = {'skip_inference': True}
        if self.compiler_kwargs is not None:
            compiler_kwargs.update(self.compiler_kwargs)
        _chainer_compiler_core.configure(**compiler_kwargs)
```
is more appropriate here. Doesn't this work for you?

Reply:
Because `graph.backward_to_with_order` depends on some global variables, `_chainer_compiler_core.configure` should be called before `graph.backward_to_with_order`.

Reply:
Yeah, I'm OK with line 242, but I think it's better to put the 4 lines here again. Otherwise, `compile` calls in line 270 and 271 will ignore `compiler_kwargs` specified by the user.

Reply:
Ah, I understood. Thank you for pointing out this part. Now I fixed.